sos/plugins/networking.py

```diff
@@ -209,6 +234,25 @@ def setup(self):
             "bridge -d vlan show"
         ])
 
+        # check if devlink module loaded and if yes, capture additional
+        # data from devlink command
+        if self.is_module_loaded("devlink"):
+            self.add_cmd_output("devlink dev show")
+            self.add_cmd_output("devlink port show")
+            self.add_cmd_output("devlink sb show")
+            self.add_cmd_output("devlink sb pool show")
+            self.add_cmd_output("devlink sb port pool show")
+            self.add_cmd_output("devlink sb tc bind show")
+            self.add_cmd_output("devlink region show")
+            self.add_cmd_output("devlink dev param show")
+            for dev in self.get_devlink_devs():
+                self.add_cmd_output("devlink -v resource show %s" % dev)
+                self.add_cmd_output("devlink dpipe header show %s" % dev)
+                devlink_table_file = self.get_cmd_output_now("devlink dpipe table show %s" % dev)
```

Comment:
Ditto - you can just pass the entire list returned by `get_devlink_devs()` into a single call, using a list comprehension to build the commands from the format string and the device list.